lib/chat_api/conversations/query.ex

```diff
@@ -0,0 +1,53 @@
+defmodule ChatApi.Conversations.Query do
```

Comment:
hmm personally I feel like this abstraction isn't necessary at this point... can we keep everything in the `Conversations` context for now?

(the reason being: if I'm looking at the methods performing these queries, it's hard to know exactly what's going on if I just see `Query.by_customer`... also, it's not like we're using these queries in multiple places yet, so seems unnecessary 🤷 )

Reply:
Ok!